cloud/gcp/modules/database/main.tf

```diff
@@ -21,12 +21,13 @@ resource "google_secret_manager_secret_iam_binding" "binding" {
   role = "roles/secretmanager.secretAccessor"
   members = [
     "user:ktoor@google.com",
-    "serviceAccount:${var.terraform_service_account}"
+    "serviceAccount:${var.terraform_service_account}",
+    "serviceAccount:${var.application_service_account_email}"
   ]
 }
 
 resource "google_sql_database_instance" "civiform_db" {
-  name             = "civiform-db-instance"
+  name             = "civiform-db-instance-3"
```

Comment:
?

Reply:
It takes a week to re-use the name once you delete a database. So I just added a random number to make things work. Will remove it when all of this goes to production

Reply:
can you add a comment in here for that!

Reply:
suggestion (blocking): use a random value generator instead e.g. https://github.com/seattle-uat/civiform/blob/ab0d52f3b037e1e2693c5798c77714fc052fcb57/cloud/azure/modules/app/main.tf#L1

Reply:
I don't think the name here will block anything in production or dev. This is just an interim issue